[E 2024-10-18 17:30:25,524 C:\Users\Clary Lin\AppData\Local\Programs\Python\Python312\Lib\site-packages\shioaji\utils.py:49:raise_resp_error] {'status': {'status_code': 400}, 'response': {'token': '', 'accounts': [], 'detail': 'ip: 180.177.3.245 not allow.', 'ip_address': '180.177.3.245', 'person_id': 'E225206498', 'client_name': '', 'contract_download': False, 'web_token': '', 'solace_props': None}}
[E 2024-10-19 05:01:43,178 C:\Users\Clary Lin\AppData\Local\Programs\Python\Python312\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
